### ISSUE TITLE
STAY AWAY from new 2018 signed firmware!!!

### ISSUE THREAD

**Issue by riogrande75** [BotoX/Dahua-Firmware-Mod-Kit#14]
I just upgraded my VTO2000A with a 2018'er signed firmware the official way. It comes along with a brand new bootloader "1.46t(DM365)18:30:31 Jan 5 2018".
For me it looks like it blocks any "inofficial" upgrade possibility now: TFTP upgrade does not work anymore and UART output is just "àààààààà...".
Can anybody confirm this?

**Comment by BotoX** [comment]
Yes, all of the newer cameras come with these changes. If you have a signed firmware with an old bootloader you might be able to downgrade.

**Comment by Creteil** [comment]
@BotoX, how I can verify the presence of the brand new bootloader "1.46t(DM365)18:30:31 Jan 5 2018" riogrande75 described in is message before upgrading ?

**Comment by riogrande75** [comment]
I have to correct my statement from above: It is still possible to install any software with the TFTP method, even with brand new 2018 bootloaders! Also UART is ok, it just happend to me because of a defective USB-serial converter.
I went from VTO2000A SIP software 20170425 to 20180105 and later to 20180627 (UBL: Aug 3 2017, UBOOT: May 23 2018). After installing a modified part of the 20180627 (data-x.cramfs.img) the system kept on rebooting every few minutes - this is due the failed signature verification in actual sw versions. After flashing a old modified 20170425 firmware via TFTP the system works fine again. Even upgrading to 20180627 via VDP-CONFIG tool was ok.

Anyhow: With little mod's in the config files the firmware-mod-kit can still handle actual firmware images but due to a missing signature creating mechanism, the tool is pretty useless now. I fear that this cannot be solved unless someone hacks this.

**Comment by Creteil** [comment]
@riogrande75 

Can you share your :"upgrade_info_7db780a713a4.txt" used for the old 20170425 firmware.

**Comment by Creteil** [comment]
@riogrande75 

Can you share your "upgrade_info_7db780a713a4.txt" for flashing the 20170425 firmware please ?

**Comment by riogrande75** [comment]
I did not create it by myself. I let a tool do it.
Pls. check [https://ipcamtalk.com/threads/dahua-ipc-easy-unbricking-recovery-over-tftp.17189/](url) for a full tutorial how to flash via TFTP.
Then edit commands txt accordingly:
`run dr ==>flash romfs-x.cramfs.img`
`run dk ==>flash kernel-x.cramfs.img`
`run du ==>flash user-x.cramfs.img`
`run dw ==>flash web-x.cramfs.img`
`run dp ==>flash data-x.cramfs.img`
`run dc ==>flash custom-x.cramfs.img`
`tftp 0x82000000 pd-x.squashfs.img; flwrite`
`tftp 0x82000000 .FLASHING_DONE_STOP_TFTP_NOW`
Off course, you have to flash all partitions that are in the .bin firmware archive. I suggest not to flash the bootloader, it has no affect on the running system und is a bit risky.

**Comment by Churiloff** [comment]
Good evening! Happy New Year! Tell me please, is it possible to flash the Chinese IPC-HFW4433M-I2 into English? Answer please! thank

**Comment by veterinator** [comment]
Hello. I purchased on Tao 6 pieces IPC-HFW4233K-AS-LED. And I was not pleasantly surprised when it turned out that the interface is only in Chinese. I began to look for a solution to this problem and came across your article in the github. Compiled the firmware, downloaded through the TFTP. The camera went into a cyclic reboot. I was able to restore it with native firmware (https://www.dahuatech.com/service/searchlist.html?keyword=DH-IPC-HFW4233K-AS-LED)
You can help create the correct mod for the firmware (https://www.dahuatech.com/service/searchlist.html?keyword=DH-IPC-HFW4233K-AS-LED) I myself could not, I do not have enough knowledge. Thank you in advance.

**Comment by jude201186** [comment]
anyone can make now English fimware for DH-IPC-HDBW4433R-ZS  2019 ?

**Comment by ghost** [comment]
Off course not unless someone cracks the signature mechanism.

**Comment by jude201186** [comment]
@sithyoda  what it is your email can you help me  ?

**Comment by jude201186** [comment]
now DH-IPC-HDW4631C-A also can not upgrade

**Comment by ghost** [comment]
I hardly recommend everybody to read this thread: [https://ipcamtalk.com/threads/dahua-ipc-easy-unbricking-recovery-over-tftp.17189/](url)
If you don't get it working again, read the thread again!

**Comment by jude201186** [comment]
can anyone tell me how to downgrade DH-IPC-HDBW4433R-ZS ? now our fimware can not upgrade dahua new chinese fimware 2018.12 but can do  fimware on 2018.08

**Comment by veterinator** [comment]
Hello. There are three cameras HFW4233K-AS-LED and two HFW4233F-ZSA with an interface in Chinese only. There are also two HFW4233F-ZSA with cracked firmware with Russian and English interfaces. Question:
1) Is it possible to pull out the firmware from the merged dump of russified cameras for further use (Russification of Chinese cameras by uploading this firmware through the web)?
2) How to edit the serial number and MAC of the camera in the merged dump in order to Russify by uploading the full dump?
Both models belong to the same series, so if it is possible to pull out the firmware, then it will fit both cameras.
